_data/JustinHuey/JH.md

```diff
@@ -0,0 +1,8 @@
+---
+name: Justin Huey
+institution: Balboa High School
+profile_pic: JH.JPG
+quote: We made it! YOLO
+github_user: JustinHuey1
+
```

Comment:
Not sure if this extra line would cause any issues, but may want to remove it just in case.